modules/gapi/src/compiler/gmodel.hpp

```diff
@@ -222,6 +222,11 @@ struct HasIntrinsics
     static const char *name() { return "HasIntrinsicsFlag"; }
 };
 
+struct DesyncCfg {
+    int index;
+    bool drop_last;
```

Comment:
Should this be here at all? You don't put any other operation's arguments to the graph metadata.

`index` here is not a config. Users don't configure indexes. `index` here is our way to distinguish different desync paths within the graph model.